pulp_ostree/app/viewsets.py

```diff
@@ -177,13 +340,108 @@ class OstreeRepositoryVersionViewSet(core.RepositoryVersionViewSet):
 
     parent_viewset = OstreeRepositoryViewSet
 
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_repository_model_or_obj_perms:ostree.view_ostreerepository",
+            },
+            {
+                "action": ["destroy"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_repository_model_or_obj_perms:ostree.delete_ostreerepository",
```

Comment:
To destroy a repository version, we should be able to access it. I am missing the view permission here. Both, for a repository version and repository itself.

This is a safe approach we can take from pulp_rpm: https://github.com/pulp/pulp_rpm/blob/486fd7ea7908c63664960322a227cdeee82b71a8/pulp_rpm/app/viewsets/repository.py#L268-L283